tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -995,8 +995,6 @@ void test_directory_entry() {
         // Test VSO-892890 "std::filesystem::directory_entry constructor initializes wrong state"
         EXPECT(!nonexistentEntryEc.exists());
         EXPECT(good(ec));
-
-        EXPECT(throws_filesystem_error([&] { nonexistentEntryEc.refresh(); }, "directory_entry::refresh", nonexistent));

```

Comment:
Removing this is a minimal way to get the test passing again, but I think it would be better to add test coverage verifying that the changed functions all accept nonexistent paths. Adding such coverage would thoroughly address @BillyONeal's concern and demonstrate that the behavior is not regressing.

The constructors are already tested above, and `assign` and `replace_filename` are already tested below, so we just need to cover `refresh()`:

https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/tests/std/tests/P0218R1_filesystem/test.cpp#L1212-L1220